src/views/UserSettings.vue

```diff
@@ -69,23 +73,41 @@ export default {
         currency: null,  // see initUserSettingsForm
       },
       currencyList: constants.CURRENCY_LIST,
-      languageList: constants.LANGUAGE_LIST,
+      languageList: [],
+      languagesData: null,
+      languageTranslationCompletion: null,
     }
   },
+  watch:{
+    'userSettingsForm.selectedLanguage': async function () {
+      console.log("selected language: ", this.userSettingsForm.selectedLanguage)
+      if (this.userSettingsForm.selectedLanguage !== null) {
+        this.languageTranslationCompletion = await localeManager.calculateTranslationCompletion(this.userSettingsForm.selectedLanguage.code)
```

Comment:
in the v2 it could be a github action instead (that stores the completion value in the `languages.json`)

Reply:
Yes definitely